src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixLegacyResourceDesignerStep.cs

```diff
@@ -20,7 +20,7 @@ namespace MonoDroid.Tuner
 	public class FixLegacyResourceDesignerStep : LinkDesignerBase
 	{
 		internal const string DesignerAssemblyName = "_Microsoft.Android.Resource.Designer";
-		internal const string DesignerAssemblyNamespace = "Microsoft.Android.Resource.Designer";
+		internal const string DesignerAssemblyNamespace = "_Microsoft.Android.Resource.Designer";
```

Comment:
Were we going to try `[EditorBrowsable (EditorBrowsableState.Never)]` on the `Resource` class, too?

Reply:
I want to test that if doesn't break anything in a "fake" project first.

Reply:
So adding the attribute does hide it from view. 

In this example `Classfoo` derives from `ClassHidden`. And I didn't see `ClassHidden` in the drop  the doing `Classfoo.`. However it is still listed when doing `ClassLibrary1.`. 

Also we get this recommendation. I'm not sure if there is a way to turn that off..

![Capture](https://user-images.githubusercontent.com/810617/212056391-9ef66092-1b67-488e-8a04-81132076b722.PNG)

Reply:
Are there ways to mark your test class is auto-generated?
```
//------------------------------------------------------------------------------
// <auto-generated>
//     This code was generated by a tool.
// </auto-generated>
//------------------------------------------------------------------------------
```
I don't know if that might suppress some of the lightbulb suggestions...

Reply:
the generated .cs files we can add a comment, but the assembly itself it all IL, so that probably won't work.